technology/cassandra/src/main/java/com/github/mizool/technology/cassandra/ClusterProducer.java

```diff
@@ -86,6 +90,18 @@ private void registerJdk8TimeCodecs(Cluster cluster)
             .register(LocalTimeCodec.instance);
     }
 
+    private SocketOptions getSocketOptions()
```

Comment:
putting `getSocketOptions()` here is arbitrary, as is the order of the other private methods. reorder them so their order equals the order of calls in `produce()`.